src/services/fetchers/CryptoComparePriceWSFetcher.ts

```diff
@@ -2,19 +2,20 @@ import {inject, injectable} from 'inversify';
 
 import CryptoCompareWSClient from '../ws/CryptoCompareWSClient.js';
 import {PairWithFreshness} from '../../types/Feed.js';
+import {FeedFetcherInterface, FeedFetcherOptions} from 'src/types/fetchers.js';
 
 @injectable()
-class CryptoComparePriceWSFetcher {
+class CryptoComparePriceWSFetcher implements FeedFetcherInterface {
   @inject(CryptoCompareWSClient) cryptoCompareWSClient!: CryptoCompareWSClient;
 
-  async apply(pair: PairWithFreshness, timestamp: number): Promise<number> {
-    const price = await this.cryptoCompareWSClient.getLatestPrice(pair, timestamp);
+  async apply(params: PairWithFreshness, options: FeedFetcherOptions): Promise<number> {
+    const price = await this.cryptoCompareWSClient.getLatestPrice(params, options.timestamp || 0);
```

Comment:
`timestamp` was required and now you make it optional, it must be required here as it was

Reply:
This suggested me that it wasn't required before: https://github.com/umbrella-network/pegasus/blob/951f6d43a0a52bb48a8bdbdcd62cdb8577951f40/src/types/fetchers.ts#L31

I can change it to required, I try also to avoid optional values.

Reply:
but there was no `?` there before, it was `apply(pair: PairWithFreshness, timestamp: number)`

Reply:
Oh, there it is sorry. I don't know what I was thinking there. I will fix it.